apps/nextjs-website/src/pages/[productSlug]/apis.tsx

```diff
@@ -0,0 +1,45 @@
+import { GetStaticPaths, GetStaticProps, GetStaticPropsResult } from 'next';
+import { Product } from '@/lib/types/product';
+
+import { getApi, getApisPaths } from '@/lib/api';
+
+import Hero from '@pagopa/pagopa-editorial-components/dist/components/Hero';
+import { ApiViewer } from '@/components/ApiViewer/ApiViewer';
+
+type Params = {
+  productSlug: string;
+};
+
+export const getStaticPaths: GetStaticPaths<Params> = () => ({
+  paths: getApisPaths() as string[],
```

Comment:
```suggestion
  paths: [...getApisPaths()],
```
if you want to avoid the cast